packages/protocol/contracts/L1/libs/LibProving.sol

```diff
@@ -256,9 +255,9 @@ library LibProving {
             } else {
                 require(
                     proofVerifier.verifyZKP({
-                        verificationKey: ConfigManager(
-                            resolver.resolve("config_manager", false)
-                        ).getValue(string(abi.encodePacked("zk_vkey_", i))),
+                        verifierId: string(
+                            abi.encodePacked("plonk_verifier_", i)
```

Comment:
Just think the id could be more related to prover settings like +maxTxs and +maxCalldatasize rather than +zkProofsPerBlock? Does this different zkProofsPerBlock lead to different circuit?

Reply:
zkProofsPerBlock is always 1 for now. In the future, if we have two independent zkEVM implementation, we can change it to 2. So it is unrelated.